File: AptDemo/app_1/src/main/java/com/wpf/aptdemo/MainActivity.java
package com.wpf.aptdemo;

import android.os.Bundle;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.wpf.apt_annotation.BindView;

/**
 * Author: feipeng.wang
 * Time:   2021/7/28
 * Description : This is description.
 */
public class MainActivity extends AppCompatActivity{
    @BindView(value = R.id.hello)
    TextView helloText;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: StudyWork/app/src/main/java/com/wpf/studywork/MainActivity.kt
package com.wpf.studywork

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.wpf.studywork.extension.startAct
import com.wpf.studywork.main.TestAct

class MainActivity : AppCompatActivity() {
    private lateinit var act: MainActivity
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        act = this
        var start = findViewById<TextView>(R.id.start)
        start.setOnClickListener {
            startAct<TestAct>(act)
        }
    }
}